_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhd-frontend/src/scripts/@rhd/js/drupal-namespace.js

```diff
@@ -133,7 +133,7 @@ app.products = {
 };
 
 app.products.downloads = {
-    "cdk" : {"windowsUrl" : "/download-manager/file/cdk-3.9.0-1-minishift-windows-amd64.exe", "macUrl" : "/download-manager/file/cdk-3.9.0-1-minishift-darwin-amd64", "rhelUrl" : "/download-manager/file/cdk-3.9.0-1-minishift-linux-amd64"}
+    "cdk" : {"windowsUrl" : "/download-manager/file/cdk-3.10.0-1-minishift-windows-amd64.exe", "macUrl" : "/download-manager/file/cdk-3.10.0-1-minishift-darwin-amd64", "rhelUrl" : "/download-manager/file/cdk-3.10.0-1-minishift-linux-amd64"}
```

Comment:
@KamiQuasi Is there any reason we cannot read this directly from Download Manager?

I'll leave @jordanpagewhite to comment, but I think this is also going to need to be updated in `rhd-frontend` as well as this file is not used by the `rhdp2` theme.

Reply:
@KamiQuasi I am also wondering why we can't just fetch this data from Download Manager. It seems very risky to have this hidden away in a file like this. Rob is right that this will only change this value in the `rhdp` theme, which is only loaded for old, paragraphs-based Product pages. That being said, the `cdk` Product page _is_ an old, paragraphs-based Product. If a content editor were to update/change this Product to the new, assembly-based approach though, this data would not be available.

It seems to me that this would best be resolved in Download Manager unless there is some compelling reason that we (developers.redhat.com) have some unique use-case that deviates from other consumers of Download Manager.